builtin/mainmenu/tab_credits.lua

```diff
@@ -26,25 +26,34 @@ local core_developers = {
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
 	"Andrew Ward (rubenwardy) <rw@rubenwardy.com>",
 	"Krock/SmallJoker <mk939@ymail.com>",
-	"Lars Hofhansl <larsh@apache.org>",
+	"Lars Hofhansl <lhofhansl@yahoo.com>",
+	"Pierre-Yves Rollo <dev@pyrollo.com>",
+	"v-rob <robinsonvincent89@gmail.com>",
 }
 
 local active_contributors = {
-	"Hugues Ross [Formspecs]",
-	"Maksim (MoNTE48) [Android]",
-	"DS [Formspecs]",
-	"pyrollo [Formspecs: Hypertext]",
-	"v-rob [Formspecs]",
-	"Jordach [set_sky]",
-	"random-geek [Formspecs]",
-	"Wuzzy [Pathfinder, builtin, translations]",
-	"ANAND (ClobberXD) [Fixes, per-player FOV]",
-	"Warr1024 [Fixes]",
-	"Paul Ouellette (pauloue) [Fixes, Script API]",
-	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com> [Audiovisuals]",
-	"HybridDog [Script API]",
-	"dcbrwn [Object shading]",
-	"srifqi [Fixes]",
+	-- TODO: add/keep areas of work here
+	"Wuzzy",
+	"Zughy",
+	"MoNTE48",
+	"numzero",
+	"Lars Müller",
+	"Desour",
+	"HybridDog",
+	"Hugues Ross",
+	"ClobberXD",
+	"Vincent Robinson",
+	"luk3yx",
+	"hecks",
+	"LoneWolfHT",
+	"Lejo",
+	"EvidenceB",
+	"Paul Ouellette",
+	"TheTermos",
+	"David CARLIER",
+	"Danila Shutov",
+	"Jean-Patrick Guerrero",
+	"k.h.lai",
```

Comment:
Combined all areas of work/contributions in a single comment for copy&paste:

```Lua
	"Wuzzy [devtest game, visual corrections]",
	"Zughy [Various fixes]",
	"MoNTE48 [Android improvements]",
	"numzero [Graphics and render fixes]",
	"appgurueu [Various internal fixes]",   -- using GitHub username here
	"Desour [Formspec and vector API changes]", -- commits found using email address
	"HybridDog [Rendering fixes and documentation]",
	"Hugues Ross [Graphics-related improvements]",
	"ClobberXD [Re-binding mouse buttons]",
	"v-rob",-- DUPLICATED: core dev
	"luk3yx [Fixes]",
	"hecks [Audiovisuals, Lua API]",
	"LoneWolfHT [Object crosshair, documentation fixes}",
	"Lejo [Server-related improvements]",
	"EvidenceB [Compass HUD element]", -- company
	"pauloue [GCC warning fix]", -- using GitHub username
	"TheTermos", --?? cannot find any commit
	"David CARLIER [UNIX & Haiku build fixes],
	"Danila Shutov",   --?? cannot find any commit
	"Jean-Patrick Guerrero [Visual fixes]",
	"k.h.lai [Memory leak fixes, documentation]",
```

Reply:
@kilbith @pyrollo how would you like to be attributed?

Reply:
`LoneWolfHT [Object crosshair, documentation fixes}` why is this closed by a curly bracket?

Reply:
@SmallJoker how come `model[]` is a “visual fix”? Please fix my attributions.

Reply:
> "TheTermos",   -- ?? cannot find any commit

https://github.com/minetest/minetest/commits/master?author=TheTermos

> "Danila Shutov",   -- ?? cannot find any commit

https://github.com/minetest/minetest/commits/master?author=dcbrwn

Reply:
Updated accordingly. I used `git log 5.3.0..HEAD --oneline --author="<>"` to filter the commit log but apparently something isn't working correctly there.

Reply:
Please update mine with `[visual improvements, various fixes]`. Visual ones: #10756 #10152